src/myfunctions.js

```diff
@@ -0,0 +1,100 @@
+import {array} from './index.js'; // eslint-disable-line
+
+const todoList = () => {
+  const form = document.querySelector('.form');
+  const list = document.createElement('div');
+  list.className = 'input-div';
+  form.appendChild(list);
+  const checkboxes = document.createElement('input');
+  checkboxes.className = 'input';
+  checkboxes.type = 'checkbox';
+  const listText = document.createElement('p');
+  listText.className = 'listContent';
+  const editBtn = document.createElement('i');
+  editBtn.className = 'fas fa-ellipsis-v';
+  const deleteBtn = document.createElement('i');
+  deleteBtn.className = 'fas fa-trash-alt icon2';
+  list.append(checkboxes, listText, editBtn, deleteBtn);
+
+  checkboxes.addEventListener('click', () => {
+    editBtn.classList.toggle('remove-icon-active');
+    deleteBtn.classList.toggle('icon2');
+    listText.classList.toggle('listContent-disable');
+    list.classList.toggle('changeBg');
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const empty = [];
+    const selectInput = document.querySelectorAll('.input-div');
+    for (let i = 0; i < getting.length; i += 1) {
+      if (selectInput[i].classList.contains('changeBg')) {
+        getting[i].completed = true;
+      } else {
+        getting[i].completed = false;
+      }
+      empty.push(getting[i]);
+      localStorage.setItem('list', JSON.stringify(empty));
+    }
+  });
+  const clearAll = document.querySelector('#clear');
+  clearAll.addEventListener('click', () => {
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const variable = document.querySelectorAll('.changeBg');
+    for (let i = 0; i < variable.length; i += 1) {
+      form.removeChild(variable[i]);
+    }
+    const empty = [];
+    for (let i = 0; i < getting.length; i += 1) {
+      if (getting[i].completed === true) {
+        continue; // eslint-disable-line
+      }
+      empty.push(getting[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
+
+  deleteBtn.addEventListener('click', () => {
```

Comment:
- [ ] A function for editing and removing task descriptions should be implemented, so your app has the functionality for adding and removing tasks